src/Redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+const CREATE = 'bookstore-app/books/CREATE';
+const REMOVE = 'bookstore-app/books/REMOVE';
+
```

Comment:
- Please, kindly remove the added `-app` attached to the `bookstore` to meet the project requirement.
`const CREATE = 'bookstore/books/CREATE'`

Reply:
- Please, kindly rename your `Redux` directory to `redux` and `Components` to `components` to meet the structure of the files in the ducks' pattern to meet the project requirement.
- Also remember to update it in your import statement.

Reply:
sure, will implement them right away

Reply:
I just tried to change the directory names to lower case but it isn't working, although, the subfolders in the Redux directory are all in lower case.